docs/threat-model.md

```diff
@@ -132,17 +142,11 @@ Authorized users with malicious intent exploiting their privileges to misuse or
 Implement proper access controls, least-privilege principles, and regular auditing to detect and prevent insider threats.
 Regularly review and monitor user activities and permissions to identify any suspicious behavior.
 
-## Lack of Input Validation for Connect to GitHub Function
-
-### Threat
+The contributions to the project are monitored by the repository owner.
 
-Insufficient input validation for the `connect_to_github` function, allowing for potentially malicious inputs.
+Users of the library are responsible within their own sphere of influence.
 
-### Mitigation
-
-Implement robust input validation and strict parsing of inputs for the `connect_to_github` function. Validate and sanitize
-user-provided inputs to mitigate potential security risks, such as ensuring the username and password inputs adhere to
-the expected format and length.
+## Lack of Input Validation for Connect to GitHub Function
```

Comment:
This turned into an empty section here.  Is that on purpose?

Reply:
This is a missed delete.  This risk was redundant with an earlier risk.

Reply:
@halprin please re-review.  this is a really strict repo, apparently

Reply:
You could just tell me to stuff it.  😉